src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,34 @@
 import React from 'react';
+import MovieCard from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export interface Films {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string
+}
```

Comment:
move it to external file

Reply:
also, why plural, it's type for one `film`
```suggestion
export interface Film {
  title: string,
  description: string,
  imgUrl: string,
  imdbUrl: string,
  imdbId: string
}
```